### PR TITLE
Ground rules

### PR DESCRIPTION
**Pull request by commuterjoy**
/cc @richard-still-ft @phamann @matthew-andrews @wheresrhys @tom-parker

### CODE REVIEW

**Comment by tom-parker** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
 This is the 'tag manager' for Next. It's to be used to house all third party scripts.
+
+## Background
+
+ft.com has an elgalitarian means of adding JavaScript to the web page.
```
Comment (wheresrhys) [inline README.md]: egal

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+Anyone (in the world) can make a PR to this project.
+
+Successful requests will be reviewed and merged in to the FT code base.
+
+### Management
+
+- Each script MUST have an expiring [feature flag](https://github.com/Financial-Times/next-flags-api) to allow it to be toggled on and off.
+- You MUST scope the loading of the tag to collect the minimally useful data set. _Eg, scope the script to a geographic region, individual URL, group of users, or use sampling techniques._
+- You MUST document what the tag does and its impact on performance, network etc. in the pull request.
```
Comment (wheresrhys) [inline README.md]: And in a comment in the code?

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+- You MUST document the browser support.
```
Comment (wheresrhys) [inline README.md]: an we require it complies with the origami spec on documenting browser support and browser features http://origami.ft.com/docs/syntax/origamijson/

File: README.md [hunk 4]
```diff
@@ -1 +1,37 @@
...
+- Requests for the addition of tag managers or anything that exists soley for the purpose of inserting other tracking code will be rejected.
+- You MUST consider whether we already collect this data in RedShift or other provider, i.e. avoid collecting it twice.
+- You MUST consider the effort/cost of adapting existing tracking systems to collect the additional information needed before introducing new libraries.
```
Comment (wheresrhys) [inline README.md]: + You SHOULD discuss your requirements and proposals directly with the next team

File: README.md [hunk 5]
```diff
@@ -1 +1,37 @@
...
+
+### Performance
+
+- 3rd party tracking and associated assets SHOULD be hosted on FT.com, or supplied as a bower module implementing semver.
```
Comment (wheresrhys) [inline README.md]: Should be on a CDN or similar highly available & distributed infrastructure?

File: README.md (hunk 1 above)
Comment (commuterjoy) [inline README.md]: i try and use a big word and spell it wrong.

File: README.md (hunk 3 above)
Comment (commuterjoy) [inline README.md]: maybe, i think lets see if anyone complies with the basics first

**Pull request merged**